sound/soc/sof/intel/hda.c

```diff
@@ -429,6 +429,10 @@ MODULE_PARM_DESC(use_msi, "SOF HDA use PCI MSI mode");
 #define hda_use_msi	(1)
 #endif
 
+int sof_hda_position_quirk = SOF_HDA_POSITION_QUIRK_USE_DPIB_REGISTERS;
+module_param_named(position_quirk, sof_hda_position_quirk, int, 0444);
+MODULE_PARM_DESC(position_quirk, "SOF HDaudio position quirk");
```

Comment:
Can we add more info in this help message , such as 0 xxx, 1xxx, 2xxx. User can get help.

Reply:
we haven't done so in the past for any kernel parameter. this isn't supposed to be friendly for regular users, so not sure about the value of being more verbose and having to track both help text and definitions.